task/01-strings-tasks.js

```diff
@@ -238,7 +243,10 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    if (typeof value === 'string' || value instanceof String) {
+        return true;
+    }
+    return false;
```

Comment:
we can just `return (typeof value === 'string' || value instanceof String)` with the same effect

Please, see https://javascript.info/logical-operators